examples/slider.js

```diff
@@ -100,6 +100,22 @@ ReactDOM.render(
       <Slider dots step={20} defaultValue={100} onAfterChange={log} dotStyle={{ borderColor: 'orange' }} activeDotStyle={{ borderColor: 'yellow' }} />
     </div>
     <div style={style}>
+      <p>Inverted Basic Slider</p>
+      <Slider inverted onChange={log} />
```

Comment:
The style of Slider had been inverted, but the value is not, you can check it locally.

And this means that you unit test need to be updated, too.

Reply:
I've check locally and seems like all work as expected.

Reply:
@GeorgeSokolovsky Noop. `Inverted` means when you move slide handler form right to left, the value goes from 0 to 100.

Reply:
@rufman Could you check this example? Seems the value is not inverted.